src/Symfony/Bundle/FrameworkBundle/Routing/DelegatingLoader.php

```diff
@@ -28,20 +27,17 @@
 class DelegatingLoader extends BaseDelegatingLoader
 {
     protected $parser;
-    protected $logger;
     private $loading = false;
 
     /**
      * Constructor.
      *
      * @param ControllerNameParser    $parser   A ControllerNameParser instance
-     * @param LoggerInterface         $logger   A LoggerInterface instance
      * @param LoaderResolverInterface $resolver A LoaderResolverInterface instance
      */
-    public function __construct(ControllerNameParser $parser, LoggerInterface $logger = null, LoaderResolverInterface $resolver)
+    public function __construct(ControllerNameParser $parser, LoaderResolverInterface $resolver)
```

Comment:
@xabbuh : Thanks for the reference. If I understand you right, this is what the changes would be like: https://github.com/symfony/symfony/compare/2.8...ogizanagi:2_8/delegating_loader_logger_deprecation
If it looks good to you, I'll create a second PR to merge before this one into 2.8.